bindata/network/openshift-sdn/sdn.yaml

```diff
@@ -91,10 +97,21 @@ spec:
           rm -f /etc/cni/net.d/80-openshift-network.conf
           cp -f /opt/cni/bin/openshift-sdn /host-cni-bin/
 
+          mtu_flag=
+          if [[ -f /config/mtu ]]; then
+            mtu_flag="--mtu $(cat /config/mtu)"
```

Comment:
There's no reason to put it into the config file and then read it out of it here; you can just do
```
mtu_flag={{if .MTU}}--mtu {{.MTU}}{{end}}
routable_mtu_flag={{if .RoutableMTU}}--routable-mtu {{.RoutableMTU}}{{end}}
```

alternatively, put the MTU and RoutableMTU into an `mtu.yaml` file in the config, and then pass `--mtu-override /config/mtu.yaml` to sdn

Reply:
I did it this way to avoid the rollout of sdn daemonset.
I will use the `mtu.yaml` approach